sdk/lib/rtl/network.c

```diff
@@ -798,152 +778,226 @@ RtlIpv6StringToAddressExA(
     return Status;
 }
 
-/*
```

Comment:
We should probably move this into a different file, like wine.c, so we can actually respect the license and put the file header from the original file there.

Reply:
It would look odd if a subset of the conversion functions were in one file, and the rest in this one.
Or all of them should be imported from Wine and avoid mixing 2 different implementations?

Reply:
It has to conform with the license though. So you need to at least add a comment here indicating where the code comes from and it's author(s). Something like this:
```
/*
 * This function was imported from wine-x.y on <date>.
 *
 * This library is free software; you can redistribute it and/or
 * modify it under the terms of the GNU Lesser General Public
 * License as published by the Free Software Foundation; either
 * version 2.1 of the License, or (at your option) any later version.
 *
 * For full text of the license see COPYING.LIB in the root of this project.
 *
 * Copyright 1996-1998 Marcus Meissner
 * Copyright 1999      Alex Korobka
 * Copyright 2003      Thomas Mertes
 *
 /
 ```

Reply:
Done.